doc/classes/MeshInstance.xml

```diff
@@ -52,13 +52,14 @@
 			<return type="Material" />
 			<argument index="0" name="surface" type="int" />
 			<description>
-				Returns the [Material] for a surface of the [Mesh] resource.
+				Returns the override [Material] for a surface of the [Mesh] resource.
+				[b]Note:[/b] This function only returns [i]override[/i] materials associated with this [MeshInstance]. Consider using [member get_active_material] or [member Mesh.surface_get_material] to get materials associated with the [Mesh] resource.
```

Comment:
These should be `[method ...]` references, not `[member ...]`. `[member ...]` is for properties.

Please fixup the commit using `git commit --amend` and a force push to update the PR, so it stays as one commit (see [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#modifying-a-pull-request) for details).

Reply:
Thank you, such a dumb mistake!